db/dataset.py

```diff
@@ -102,15 +105,19 @@ def get(id):
             return None
 
 
-def get_public_datasets(status_value, offset, limit):
-    result = ''
+def get_public_datasets(status, offset, limit):
+    if status == "all":
+        statuses = db.dataset_eval.VALID_STATUSES
+    elif status in db.dataset_eval.VALID_STATUSES:
+        statuses = [status]
+    else:
+        raise Exception("Unknown status")
```

Comment:
thanks for the reminder. I just put `Exception` in as a temporary value, but ValueError is perfect for this.